spec/rubocop/cop/style/copyright_spec.rb

```diff
@@ -12,7 +12,6 @@ def expect_no_copyright_offense(cop, source)
 def expect_copyright_offense(cop, source)
   inspect_source(cop, source)
   expect(cop.offenses.size).to eq(1)
-  expect(cop.messages[0]).to eq(cop.message)
```

Comment:
I removed this test case, as it's essentially forcing a private method to be public in order to assert that `true == true`. 😅